ui.apps/src/main/content/jcr_root/apps/core/fd/components/commons/v1/clientlibs/layouts/list/css/list.less

```diff
@@ -0,0 +1,93 @@
+.cmp-portallister__item--list {
```

Comment:
How can list be a modifier ? Shouldn't this be cmp-portallister__item-list ? Can you please check all such usages and fix them before creating a new PR